src/profile/components/multiItemRow/MultiItemRow.tsx

```diff
@@ -0,0 +1,122 @@
+import { TextInput } from 'hds-react';
+import React from 'react';
+import { Field, Formik, FormikProps, Form } from 'formik';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+import styles from './multiItemRow.module.css';
+import { EditDataValue, EmailValue, PhoneValue } from '../../helpers/editData';
+import { createFormFieldHelpers } from '../../helpers/formik';
+import { phoneSchema, emailSchema } from '../../../common/schemas/schemas';
+import FormButtons from '../formButtons/FormButtons';
+import EditButtons from '../editButtons/EditButtons';
+import commonFormStyles from '../../../common/cssHelpers/form.module.css';
+import SaveIndicator from '../saveIndicator/SaveIndicator';
+import { useCommonEditHandling } from '../../hooks/useCommonEditHandling';
+import { RowItemProps } from '../multiItemEditor/MultiItemEditor';
+import { getFormFields } from '../../helpers/formProperties';
+
+type EmailAndPhoneFormikValue = { value: string };
+
+function MultiItemRow(props: RowItemProps): React.ReactElement {
```

Comment:
I fear there can be a risk that combining phone and email to same row component might not scale in the long run in terms of readability of the code. It makes code more abstractly named, and there's going to be places where you have to check the data-type, which increases the complexity.

Reply:
IMHO, data-type checks are easier to follow than making two files (EmailRow and PhoneRow) and then more files for common code of those two. The need for scaling this file is an abstract scenario. If there are new features, it is easier to add them here?

Currently there are 3 places where dataType is checked and 2 where it is passed as argument.
There are 122 lines of code and only those 3 lines would not be common if split in to multiple files.